### ISSUE TITLE
Dynamic Merchant Page OG Image

### ISSUE THREAD

**Issue by dadofsambonzuki** [teambtcmap/btcmap.org#146]
The OG image for the merchant page should either:

a) be specific to the merchant category;

b) be dynamically generated with a pin on the map, perhaps with the pin representing the merchant category.

**Comment by dadofsambonzuki** [comment]
I think I prefer b) as it's more useful and easier to implement and maintain.

**Comment by secondl1ght** [comment]
Yeah we can dynamically generate the location of the merchant with a pin on the map - I guess the question is, should there be other elements incorporated into the design like the merchant's name etc. Or just have a full size image of the map?

**Comment by dadofsambonzuki** [comment]
Yeah, the title is already enough I think.

Just a map with the usual merchant type specific pin.

This might help: https://docs.stadiamaps.com/static-maps/

There are options in there for caching and also for labelling (if we want to see what a pin label looks like easily).

**Comment by secondl1ght** [comment]
Ok sounds like we will keep this pretty simple. If we want the style and pin to match what users see on the main map I think we will have to do this in-house. If we don't care about it showing the same pin and icon we could use something like Stadia.

**Comment by dadofsambonzuki** [comment]
Pin should represent merchant category like the main map.

**Comment by dadofsambonzuki** [comment]
@bubelov has created our own one now as we migrate from Stadia.

This is here to remind us that we should have merchant type specific pins.

**Comment by dadofsambonzuki** [comment]
I think we can close this as we now have b), albeit without the category specific icon.

Could raise a new issue in the API repo for that.
